assets/js/search.js

```diff
@@ -138,12 +139,13 @@ const highlight = (content, term) => {
   }
 
   const resultToHTML = ({ url, title, content, term }) => {
+    const regex = /\[![a-zA-z]\][-\+]? /;
     const text = removeMarkdown(content)
     const resultTitle = highlight(title, term)
     const resultText = highlight(text, term)
     return `<button class="result-card" id="${url}">
         <h3>${resultTitle}</h3>
-        <p>${resultText}</p>
+        <p>${resultText.replace(regex, "")}</p>
```

Comment:
`removeMarkdown` here should be sufficient to strip that regex right?

Reply:
Checked this, and you're right, it is not needed.